bin/looker-resources-index/README.md

```diff
@@ -0,0 +1,18 @@
+# Looker Resources Index
+
+This folder contains data (in code) about Looker-related resources and scripts to maintain and repackage that data for publication.
+
+The main location that the data is maintained is in `src/resource-data/resources.ts`
+
+The main output of these scripts is a JSON file in `../../docs/resources` intended to be checked in to git and consumed via CDN. (However, you could conceivably also consume the typescript exports from this folder)
```

Comment:
`(However, you could conceivably also consume the typescript exports from this folder)` -> `If you need the types and LocaleStrings, you need to import the typescript exports from this folder, bin/looker-resources-index`